src/oracle/mod.rs

```diff
@@ -0,0 +1,101 @@
+use std::collections::HashMap;
+
+use testcontainers::{core::WaitFor, Image};
+
+const DEFAULT_IMAGE_NAME: &str = "gvenzl/oracle-free";
+const DEFAULT_IMAGE_TAG: &str = "23-slim-faststart";
+
+/// Module to work with [`Oracle Database`] inside of tests.
+/// The default image is [`gvenzl/oracle-free:23-slim-faststart`] (unofficial).
```

Comment:
I see mention about unsupported ARM:

> Currently, there is no Oracle Database Free port for ARM chips, hence Oracle Database Free images cannot run on the new Apple M chips via Docker Desktop.

I think this is quite important limitation, so I'd prefer to add `cfg` with `not(any(target_arch="arm", target_arch="aarch64"))` for `mod oracle_free`. Moreover, tests of modules itself won't work at all on ARM-64. 

And of course, it's better to mention that as part of documentation.